style.css

```diff
@@ -90,3 +96,576 @@ p {
   display: flex;
   justify-content: space-between;
 }
+
+.works {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  margin: 80px auto;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.grid-container {
```

Comment:
- [ ] KIndly make sure your page is not overlapping as you can see :point_down: 
![Screenshot from 2022-09-29 19-14-03](https://user-images.githubusercontent.com/97100091/193100339-032a7c57-d30b-41f4-a105-d9825cbe7db5.png)